### DISCUSSION TITLE
using of an LCD display and card reader

### DISCUSSION THREAD

**Discussion by SerikNET** [arkypita/LaserGRBL#2237] [Ideas]
Dear Arkypita,
I wonder if it is possible to add using of an LCD display and card reader into the Arduino software (like the principle of a 3D printer). Thank you in advance.
Sincerely,
Serhii Hudz

**Comment by arkypita** [comment]
The Arduino software, called "grbl" it is not LaserGRBL, and it is not the software I have developed. 

LaserGRBL run on PC. 

About your question, grbl running almost all the resources available from the Arduino chip atmega328. If you want to add any feature, including LCD or more, you should use a different port of grbl desired for more modern microcontroller, or you should run your LCD driven by a different Arduino that interface to the one running grbl
